src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldDocumentProperties.class.php

```diff
@@ -26,16 +24,20 @@ public function GetHTML()
         $sDownloadURL = $oDownloadItem->getBackendDownloadLink(true);
 
         $html = '<input type="hidden" id="'.TGlobal::OutHTML($this->name).'" name="'.TGlobal::OutHTML($this->name).'" value="'.TGlobal::OutHTML($this->data).'" />';
-        $html .= '<table border="0" class="table table-sm table-striped">
+
+        $downloadIcon = $oDownloadItem->getDownloadHtmlTag(false, true, true);
+        $html .= '<table class="table table-sm table-striped">
         <tr>
```

Comment:
Could this html also go to the Twig?

Reply:
Yes, but not with this PR. This would be a major change in this field type.